### ISSUE TITLE
Choose your theme panel silent with screen reader

### ISSUE THREAD

**Issue by jbicha** [canonical/ubuntu-desktop-installer#2346]
### What happened?

None of the widgets in the "Choose your theme" panel except for previous and text are read with a screen reader

### What was expected?

The light/dark and color buttons should be read with the screen reader

### Steps to reproduce

- Boot the installer ISO
- After it plays the welcome sound, press Super+Alt+S to turn on the screen reader
- When you get to the "Choose your theme" screen, use the arrow keys and tab keys to navigate around this screen.

### Additional context

20230920 Ubuntu Desktop Mantic ISO
